src/redux/tests/Filter.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
-import { filteredCurrencyLists } from '../currencies/currencySlice.js';
+import { filteredCurrencyLists } from '../currencies/currencySlice';
 
 const Filter = () => {
```

Comment:
- [ ] Hello.  I found out that you have created a group of files inside of the test folder, and you tested them. These files in the test folder do not link to your app. The work you did is not what the project requirements are saying. 
Here are the requirements for testing:
##### **Testing**

1. App has unit tests for several components of the app and some pure functions in Redux code.
1. App has integration tests using React Testing Library
   * Student can perform a mocking data fetching and Redux Store connection.
 
👉 I will advise you to follow the requirements as it is